corehq/apps/reports/dispatcher.py

```diff
@@ -119,6 +121,11 @@ def get_report(self, domain, report_slug, *args):
         """
         return self.get_reports_dict(domain).get(report_slug, None)
 
+    @quickcache(['domain', 'report_slug'], timeout=60)
```

Comment:
that's correct, the only reason is to not lose my position in the travis queue :trollface: (i can just use ci skip)